catalog/spatial/registry/registry-api-impl/src/main/java/org/codice/ddf/registry/api/impl/RegistryStorePublisher.java

```diff
@@ -210,7 +210,7 @@ public void handleEvent(Event event) {
                 (RegistryStore) bundleContext.getService(((ServiceEvent) event.getProperty(
                         EventConstants.EVENT)).getServiceReference());
 
-        if (!previousAutoPush && registryStore.isAutoPush()) {
+        if (registryStore.isAutoPush()) {
```

Comment:
❓ Since it doesn't look like the `Boolean` stored in the `registryStoreMap` was only used for this, could we simplify the code a little by replacing the `Map` with a `Set` that contains the list of pids we know about?